config/initializers/obis_setup.rb

```diff
@@ -25,6 +25,9 @@
   ADMIN_MAIL_TO                             = application_config['admin_mail_to']
   EPIC_RIGHTS_MAIL_TO                       = application_config['approve_epic_rights_mail_to']
   FEEDBACK_MAIL_TO                          = application_config['feedback_mail_to']
+  CONTACT_US_MAIL_TO                        = application_config['contact_us_mail_to']
```

Comment:
Riffing of off Chris's comment, will the app break if these aren't defined in application.yml?